rasa_nlu/featurizers/mitie_featurizer.py

```diff
@@ -45,7 +50,7 @@ def features_for_tokens(self, tokens, feature_extractor):
         for token in tokens:
             vec += feature_extractor.get_feature_vector(token)
         if tokens:
-            return vec / len(tokens)
+            return old_div(vec, len(tokens))
```

Comment:
In this specfic case we can use `/`. With `//` we would get a truncated float, so `//` and `old_div` are not exactly the same